MantleTests/MTLModelDifferenceSpec.m

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 Lightricks. All rights reserved.
+// Created by Maxim Grabarnik.
+
+#import <Mantle/Mantle.h>
+#import <Nimble/Nimble.h>
+#import <Quick/Quick.h>
+
+#import "MTLModel+Difference.h"
+#import "MTLTestModel.h"
+
+QuickSpecBegin(MTLModelDifferenceSpec)
+
+it(@"should raise on mismatching argument type", ^{
+	MTLTestModel *model = [[MTLTestModel alloc] init];
+	expect(model).notTo(beNil());
```

Comment:
Optional:
Personally I don't like this expect, cause it doesn't test anything. IMO it should be an assertion.
Same in all test below.